src/vshaxe/Main.hx

```diff
@@ -106,6 +108,27 @@ class Main {
 		maybeStartServer();
 	}
 
+	function setLanguageConfiguration():Void {
+		// based on https://github.com/microsoft/vscode/blob/bb02817e2e549fd88710d0e0a0336b80648e90b5/extensions/typescript-language-features/src/features/languageConfiguration.ts#L15
+		var defaultWordPattern = "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\@\\#\\%\\^\\&\\*\\(\\)\\-\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s]+)";
+		var wordPattern = defaultWordPattern + "|(@:\\w*)"; // metadata
```

Comment:
I guess this is revived from my old indentationRules branch? I don't think it's needed (anymore?) though, and `wordPattern` is optional, so I think we can just omit it.

Reply:
https://github.com/vshaxe/vshaxe/commit/3d33ec19203872b13abfad5181ed521fbe8d5c80